### PR TITLE
Updated doc about adding TUID using java data provider

### PR DESCRIPTION
**Pull request by akamarouski**

### CODE REVIEW

File: docs/advanced/dataprovider.md [hunk 1]
```diff
@@ -103,3 +103,72 @@ public class DataprovidersSampleTest implements IAbstractTest {
...
+
+you should do the following:
+
+1. Add to the data provider field with test unique identifier (TUID), for example:
+
+```
+@DataProvider(name = "dataProvider")
+public static Object[][] dataprovider() {
+    return new Object[][]{
+        {"TUID: Data1", "some data", ... },
```
Comment (vdelendik) [inline docs/advanced/dataprovider.md:148]: remove `TUID: ` from the example keeping only real keys

File: docs/advanced/dataprovider.md [hunk 2]
```diff
@@ -103,3 +103,72 @@ public class DataprovidersSampleTest implements IAbstractTest {
...
+        {"TUID: Data2", "some data", ... },
+        {"TUID: Data3", "some data", ... }
+    };
+}
+```
+
+where `"TUID: Data1"` - is the test unique identifier (`TUID:` prefix is optional, you can write just `Data1`).
```
Comment (vdelendik) [inline docs/advanced/dataprovider.md:155]: no need to add info about optional prefix.

File: docs/advanced/dataprovider.md (hunk 1 above)
Comment (akamarouski) [reply docs/advanced/dataprovider.md]: done

File: docs/advanced/dataprovider.md (hunk 2 above)
Comment (akamarouski) [reply docs/advanced/dataprovider.md]: done

**Pull request merged**